src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportJobRecord.cs

```diff
@@ -20,6 +20,7 @@ public class ExportJobRecord : JobRecord
         public ExportJobRecord(
             Uri requestUri,
             ExportJobType exportType,
+            string exportFormat,
```

Comment:
>exportFormat [](start = 19, length = 12)

Is it ok to pass null here? if yes, why it's not optional parameter like others below.
If it's no ok, can EnsureArg be added?

Reply:
Good catch, this shouldn't allow null here. I'll add an EnsureArg.